sphinxext/mongodb_conf.py

```diff
@@ -332,8 +332,8 @@
         'callable': False
     },
     {
-        'name': 'k8s-op-om-setting',
-        'tag': 'k8s-op-om-setting',
+        'name': 'omk8sop',
```

Comment:
How about 

- opsmgrkube  or
- opsmgrk8s  or

The reason I kind of prefer kube is so that when typing you don't have to reach for the number.  but I don't actually care as I won't need to use this settings, but definitely think about typing/usability.